src/sider/oppfolgingsplan/container/OppfoelgingsPlanerOversiktContainer.tsx

```diff
@@ -33,6 +33,10 @@ export const OppfoelgingsPlanerOversiktContainer = () => {
   const inaktivePlaner = oppfolgingsplaner.filter(
     (plan) => !aktivePlaner.includes(plan)
   );
+  const isNoPlans =
```

Comment:
`hasNoPlans` kanskje?

Reply:
Ikke supersterke meninger, men føler `is…` leser bedre. Jeg tenker litt sånn "Vi ser på en person og her er det ingen planer. Det er ikke personen som har planene, de ligger på/hos vedkommende" -om det gir mening 😅 
Åpen for å endre om flere tenker `has…` treffer bedre 😁

Reply:
Tenker også mest på det som at vi har ingen oppfølgingsplaner for denne personen dvs `has`, men ikke superviktig :smile:

Reply:
Da blir det `has`! 🤝🚀